include/bout/paralleltransform.hxx

```diff
@@ -54,185 +51,4 @@ protected:
   Mesh &mesh; ///< The mesh this paralleltransform is part of
 };
 
-
-/*!
- * This class implements the simplest form of ParallelTransform
- * where the domain is a logically rectangular domain, and 
- * yup() and ydown() refer to the same field.
- */
-class ParallelTransformIdentity : public ParallelTransform {
```

Comment:
I know @d7919 has feelings on this, but I'm not overly keen on moving these types out of the public API and into the private API. From the point of view of the library, it doesn't matter, and we should treat all `ParallelTransform`s the same. However, from the point of view of the user, they are different beasts that may need to be treated differently (e.g. we need zShift to create a `ShiftedMetric`).

Reply:
OK. No strong feelings from me about it. I'll put them back in `paralleltransform.hxx`.